spec/update_pull_requests_milestone_spec.rb

```diff
@@ -0,0 +1,153 @@
+  end
+
+  context 'when providing a single PR number' do
+    it 'updates the milestone of a single PR' do
+      expect(client).to receive(:update_issue).with(test_repo, 1337, { milestone: 123 })
+
+      result = run_described_fastlane_action(
+        github_token: test_token,
+        repository: test_repo,
+        pr_number: 1337,
+        to_milestone: '12.3'
+      )
+
+      expect(result).to eq([1337])
+    end
+
+    it 'removes the milestone if to_milestone is nil' do
+      expect(client).to receive(:update_issue).with(test_repo, 1337, { milestone: nil })
+
+      result = run_described_fastlane_action(
+        github_token: test_token,
+        repository: test_repo,
+        pr_number: 1337,
+        to_milestone: nil
+      )
+
+      expect(result).to eq([1337])
+    end
+  end
+
+  context 'when providing a source milestone' do
+    it 'updates the milestone of all matching and still-opened PRs' do
+      allow(client).to receive(:search_issues)
+        .with(%(repo:#{test_repo} type:pr milestone:"#{mock_milestone(12.2)[:title]}" is:open))
+        .and_return({ items: [101, 103].map { |n| mock_pr(n) } })
+
+      expect(client).to receive(:update_issue).with(test_repo, 101, { milestone: 123 })
+      expect(client).to receive(:update_issue).with(test_repo, 103, { milestone: 123 })
+
+      result = run_described_fastlane_action(
+        github_token: test_token,
+        repository: test_repo,
+        from_milestone: '12.2',
+        to_milestone: '12.3'
+      )
+
+      expect(result).to eq([101, 103])
+    end
+
+    it 'adds a PR comment if one is provided' do
+      comment = 'Updated milestone from 12.2 to 12.3'
+      allow(client).to receive(:search_issues)
+                         .with(%(repo:#{test_repo} type:pr milestone:"#{mock_milestone(12.2)[:title]}" is:open))
+                         .and_return({ items: [101, 103].map { |n| mock_pr(n) } })
+      allow(client).to receive(:issue_comments).and_return([])
+
+      expect(client).to receive(:update_issue).with(test_repo, 101, { milestone: 123 })
+      expect(client).to receive(:add_comment).with(test_repo, 101, /<!-- REUSE_ID: .* -->#{comment}/)
+      expect(client).to receive(:update_issue).with(test_repo, 103, { milestone: 123 })
+      expect(client).to receive(:add_comment).with(test_repo, 103, /<!-- REUSE_ID: .* -->#{comment}/)
+
+      result = run_described_fastlane_action(
+        github_token: test_token,
+        repository: test_repo,
+        from_milestone: '12.2',
+        to_milestone: '12.3',
+        pr_comment: comment
+      )
+
+      expect(result).to eq([101, 103])
+    end
+
+    it 'does not add a PR comment if comment is empty' do
+      allow(client).to receive(:search_issues)
+                         .with(%(repo:#{test_repo} type:pr milestone:"#{mock_milestone(12.2)[:title]}" is:open))
+                         .and_return({ items: [101, 103].map { |n| mock_pr(n) } })
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/release-toolkit/blob/a45e842d8fb56f7fa70e4634b285dd8001a01adf/spec/update_pull_requests_milestone_spec.rb#L101"></span>Use 2 (not 19) spaces for indenting an expression spanning multiple lines.</td>
    </tr>
  </tbody>
</table>